tt_metal/programming_examples/eltwise_binary/eltwise_binary.cpp

```diff
@@ -351,7 +351,7 @@ int main(int argc, char **argv) {
         TT_THROW("Test Failed");
     }
 
-    TT_FATAL(pass);
+    TT_FATAL(pass, "Error");
```

Comment:
remove

Reply:
These were added by sed. I did not see the surrounding code. Removing. Along with the other ones you pointed out.